### PR TITLE
Notification channel for Android >= 8.0

### PR DESCRIPTION
**Pull request by vitalyster**
* Google Play requires now targetSdkVersion >= 26
* Notification channels are required to display notifications on newer target API levels

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -17,11 +17,10 @@ repositories {
 }
 
 android {
-	compileSdkVersion 23
-	buildToolsVersion '28.0.1'
```
Comment (mimi89999) [inline build.gradle:21]: Why did you remove them?

File: build.gradle [hunk 2]
```diff
@@ -4,7 +4,7 @@ buildscript {
 		google()
 	}
 	dependencies {
-		classpath 'com.android.tools.build:gradle:3.1.3'
+		classpath 'com.android.tools.build:gradle:3.2.1'
```
Comment (mimi89999) [inline build.gradle:7]: It should be updated to support more recent versions of Gradle.

**Comment by mimi89999** [comment]
Works as expected on LineageOS 17.1. Was it tested on Android < 8.0?

File: example/project.properties [hunk 3]
```diff
@@ -9,4 +9,4 @@
 
 android.library.reference.1=../
 # Project target.
-target=android-23
+target=android-28
```
Comment (mimi89999) [inline example/project.properties:12]: Did you forget to update `../project.properties` or is it intended?

File: example/AndroidManifest.xml [hunk 4]
```diff
@@ -6,7 +6,7 @@
       
     <uses-sdk
```
Comment (mimi89999) [inline example/AndroidManifest.xml:7]: Is that needed?

**Comment by ge0rg** [comment]
Merged in e7d5b0981dc42845206272b4a61b789a0e653617.
